tests/Feature/StartTest.php

```diff
@@ -3,11 +3,16 @@
 namespace Tests\Feature;
 
 use Tests\TestCase;
+use App\CommandExecutor;
 
 class StartTest extends TestCase
 {
     public function testStart()
     {
+        resolve(CommandExecutor::class)
+            ->shouldReceive('getOutputBuffer')
+            ->andReturn('18.09', '1.24');
```

Comment:
Here we wanna make sure the method returns valid versions, right? So maybe use `CheckDockerVersion:: DOCKER_MIN_VERSION` so future updates continue working without having to change it here.